roles/openshift_version/meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_inventory
```

Comment:
They both depend on the facts being correct before they execute.
I'm certainly open to a more descriptive name. Normalization, defaults, and validation is what I have in mind; not sure if there might be yet more we would want to do up front with the inventory before other roles execute.

For some reason github isn't letting me respond to the previous question about `openshift_inventory` getting called multiple times. http://docs.ansible.com/ansible/playbooks_roles.html#role-dependencies says "By default, roles can also only be added as a dependency once - if another role also lists it as a dependency it will not be run again."